### DISCUSSION TITLE
Summary info provided from BRAVO defers from the one from DAX STUDIO

### DISCUSSION THREAD

**Discussion by s3r3nd1p1t1** [sql-bi/Bravo#803] [General]
Hi, I'm using v 1.0.8 and I've noticed that number of columns detected from the model is different from what I get in DAX Studio. 
The file *.vpax imported in BRAVO is the one exported from DAX Studio
_Example_ 
**Dax Studio :** 
Total Size in Memory: 61,48 MB
Tables: 20
Columns: 153
**Bravo:**
Total Size: 60,98 MB
Tables: /
Columns: 111 columns, 40 of which are not referenced within the model. 

Actually I believe that 40 are not included in the 111, indeed 111+40 is 151 which is much closer to 153, but still 2 missing. 
Also Total size is slightly different..

any of you guys know the explanation on those differences?

Thank you in advance!

**Comment by albertospelta** [comment]
The difference in the total size you're observing is due to the way each tool calculates this value. Bravo calculates the total size by considering only the sizes of columns, including data size, dictionary size, and hierarchy size for each column. On the other hand, DAX Studio includes not only the sizes of columns but also the sizes of relationships and custom hierarchies (user hierarchies). In your case, 61.48 MB (DAX Studio) - 60.98 MB (Bravo) = 0.5 MB, which corresponds to the size of the relationships and user hierarchies in your model.

Regarding the difference in the number of columns, could you let me know which version of DAX Studio you used to extract the VPAX file?

**Reply by s3r3nd1p1t1** [reply]
Hi Alberto, thanks for answering so fast. 
I would have bet on it there was an explanation for the model size difference, thanks for explaining. 

The version of DAX Studio is **3.0.11**

**Reply by albertospelta** [reply]
The difference in the number of columns is because Bravo doesn't include the [Auto Date/Time](https://learn.microsoft.com/en-us/power-bi/transform-model/desktop-auto-date-time) tables (and columns), while DAX Studio does. 
![image](https://github.com/user-attachments/assets/ece63f18-4616-4ce8-9d0e-33fd7792a6aa)

**Reply by s3r3nd1p1t1** [reply]
Thank you again Alberto! Do I understand correctly, this is unrelated to the version of DAX Studio I'm using, right?

**Reply by albertospelta** [reply]
Yes, that's correct. This is unrelated to the version of DAX Studio you're using. DAX Studio is a more advanced tool compared to Bravo and displays all the information present in the VPAX file. Bravo, on the other hand, currently displays only a subset of that information.
